src/router.ts

```diff
@@ -181,10 +181,10 @@ export class Router {
             return;
           }
 
-          const currentPrice = getPythPrice(new PublicKey(tokenToSplMint(dip_deposit.splToken)));
-          const fractionOfYear = (Date.now() - dip_deposit.expirationMs) / 365 * 24 * 60 * 60 * 1_000;
-          const vol = THEO_VOL_MAP[dip_deposit.splToken] * (1.15 + Math.random() / 10);
-          const thresholdPrice = blackScholes(currentPrice, dip_deposit.strike / 1_000_000, fractionOfYear, vol, 0.01, 'call');
+          const currentPrice = await getPythPrice(new PublicKey(tokenToSplMint(dip_deposit.splToken)));
```

Comment:
This logic really shouldnt be duplicated. Should only exist in one place.

Reply:
Ya will try to do this soon. Want to just get comfortable that the routing pricing is good first. We hit another NaN this morning and routed I assume off of an undefined pyth price. Changes going into catch these